### PR TITLE
V3 monitoring jobs - add timestamp data to exception thrown

### PR DESCRIPTION
**Pull request by scottbommarito**
This adds more debuggability to these issues.

### CODE REVIEW

File: src/NuGet.Services.Metadata.Catalog.Monitoring/NuGet.Services.Metadata.Catalog.Monitoring.csproj [hunk 1]
```diff
@@ -155,7 +155,7 @@
     <Compile Include="Utility\NullableNuGetVersionConverter.cs" />
     <Compile Include="Utility\LogEvents.cs" />
     <Compile Include="Utility\ILoggerFactoryExtensions.cs" />
-    <Compile Include="Utility\SafeExceptionConverter.cs" />
+    <Compile Include="Utility\SafeExceptionConverter.cs" /
```
Comment (scottbommarito) [inline src/NuGet.Services.Metadata.Catalog.Monitoring/NuGet.Services.Metadata.Catalog.Monitoring.csproj]: woops

Comment (chenriksson) [inline src/NuGet.Services.Metadata.Catalog.Monitoring/NuGet.Services.Metadata.Catalog.Monitoring.csproj]: accident?

Comment (scottbommarito) [reply src/NuGet.Services.Metadata.Catalog.Monitoring/NuGet.Services.Metadata.Catalog.Monitoring.csproj]: Yup, just fixed it.

I resolved a merge conflict using GitHub UI and it appears to have been the culprit

File: src/NuGet.Services.Metadata.Catalog.Monitoring/Validation/Test/Exceptions/TimestampComparisonException.cs [hunk 2]
```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Newtonsoft.Json;
+
+namespace NuGet.Services.Metadata.Catalog.Monitoring
+{
+    public class TimestampComparisonException : Exception
```
Comment (chenriksson) [inline src/NuGet.Services.Metadata.Catalog.Monitoring/Validation/Test/Exceptions/TimestampComparisonException.cs]: Maybe this should extend ValidationException?

**Pull request merged**